apps/www/pages/partners/experts/[slug].tsx

```diff
@@ -160,7 +160,7 @@ function Partner({
                   <div className="flex items-center justify-between py-2">
                     <span className="text-lighter">Category</span>
                     <Link href={`/partners/experts#${partner.category.toLowerCase()}`}>
-                      <a className="text-brand hover:text-brand-300 transition-colors">
+                      <a className="text-brand hover:text-underline transition-colors">
```

Comment:
you could use `text-brand-600`

we've recently changed the tailwind classes

Reply:
on the hover text or on the default text? where should i use text-brand-600